Updated BRMS and BPM references

emmurphy1 opened the pull request.

mportman12 reviewed (approved):
LGTM

lahinson reviewed (approved):
lgtm

bergerhoffer commented on supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc:
```diff
@@ -139,11 +139,11 @@
 [[realtime-decision-server]]
 ==== image:images/yes.png[yes] Realtime Decision Server (noun)
-*Description*: In Red Hat JBoss BRMS and Red Hat JBoss BPM Suite, the _Realtime Decision Server_ is a standalone, built-in component that can be used to instantiate and execute rules through interfaces available for REST, JMS, or a Java client-side applications. Created as a web deployable WAR file, this server can be deployed on any web container. The current version of the Realtime Decision Ser ...
+*Description*: In Red Hat Process Automation Manager, the _Realtime Decision Server_ is a standalone, built-in component that can be used to instantiate and execute rules through interfaces available for REST, JMS, or a Java client-side applications. Created as a web deployable WAR file, this server can be deployed on any web container. The current version of the Realtime Decision Server is inclu ...
```
Can you remove "both" from the last sentence (it doesn't make sense now that there is just one listed).

bergerhoffer reviewed (commented):
Just one nit if you don't mind adjusting before we merge. Thanks again!

bburt-rh commented on supplementary_style_guide/glossary_terms_conventions/general_conventions/b.adoc:
```diff
@@ -21,11 +21,11 @@
 [[backward-chaining]]
 ==== image:images/yes.png[yes] backward chaining (verb)
-*Description*: In Red Hat JBoss BRMS and Red Hat JBoss BPM Suite, _backward chaining_ is a feature of the rule engine. The backward chaining process is often referred to as derivation queries. It is not as common compared to reactive systems because Red Hat JBoss BRMS is primarily reactive forward chaining, that is, it responds to changes in your data. The backward chaining added to the rule engi ...
+*Description*: In Red Hat Process Automation Manager, _backward chaining_ is a feature of the rule engine. The backward chaining process is often referred to as derivation queries. It is not as common compared to reactive systems because Red Hat Process Automation Manager is primarily reactive forward chaining, that is, it responds to changes in your data. The backward chaining added to the rule  ...
```
```suggestion
*Description*: In Red Hat Process Automation Manager, _backward chaining_ is a feature of the rule engine. The backward chaining process is often referred to as _derivation queries_. It is not as common compared to reactive systems because Red Hat Process Automation Manager is primarily reactive forward chaining, that is, it responds to changes in your data. The backward chaining added to the rule engine is for product-like derivations.
```
Since `derivation queries` is presented as an alernative phrase, I think it should italicized. What do you think?

bburt-rh commented on supplementary_style_guide/glossary_terms_conventions/general_conventions/k.adoc:
```diff
@@ -309,11 +309,11 @@ Do not capitalize the first letter.
 [[knowledge-store]]
 ==== image:images/yes.png[yes] knowledge store (noun)
-*Description*: In Red Hat JBoss BRMS and Red Hat JBoss BPM Suite, _knowledge store_ is a centralized repository for your business knowledge. The knowledge store connects to the Git repository to store various knowledge assets and artifacts at a single location.
+*Description*: In Red Hat Process Automation Manager, _knowledge store_ is a centralized repository for your business knowledge. The knowledge store connects to the Git repository to store various knowledge assets and artifacts at a single location.
```
```suggestion
*Description*: In Red Hat Process Automation Manager, _knowledge store_ is a centralized repository for business knowledge. The knowledge store connects to a Git repository to store various knowledge assets and artifacts at a single location.
```

bburt-rh commented on supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc:
```diff
@@ -513,6 +490,19 @@ Always spell out the full product name of the host, and do not capitalize the te
+[discrete]
+[[red-hat-process-automation-manager]]
+==== image:images/yes.png[yes] Red Hat Process Automation Manager (noun)
+*Description*: _Red Hat Process Automation Manager_ is the Red Hat platform for Business Process Management (BPM). Red Hat Process Automation Manager enables enterprise business and IT users to document, simulate, manage, automate, and monitor business processes and policies. It is designed to empower business and IT users to collaborate more effectively, so business applications can be changed m ...
```
```suggestion
*Description*: _Red Hat Process Automation Manager_ is the Red Hat platform for Business Process Management (BPM). By using Red Hat Process Automation Manager, enterprise business and IT users can document, simulate, manage, automate, and monitor business processes and policies. The platform is designed to empower business and IT users to collaborate more effectively, so business applications can be changed more easily and quickly.
```
- Per ISG, avoid anthropomorphic phrasing such as saying that a system "enables" or "allows" humans to to do things. https://www.ibm.com/docs/en/ibm-style?topic=grammar-anthropomorphism
- `It` in the last sentence has an unclear antecedent.

bburt-rh commented on supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc:
```diff
@@ -1079,23 +1069,23 @@ Use the name of the tab when you refer to it, for example, "the *Storage* tab".
 [[rule]]
 ==== image:images/yes.png[yes] rule (noun)
-*Description*: In Red Hat JBoss BRMS and Red Hat JBoss BPM Suite, a _rule_ provides the logic for the rule engine to execute against. A rule includes a name, attributes, a “when” statement on the left side of the rule, and a “then” statement on the right side of the rule.
+*Description*: In Red Hat Process Automation Manager, a _rule_ provides the logic for the rule engine to execute against. A rule includes a name, attributes, a “when” statement on the left side of the rule, and a “then” statement on the right side of the rule.
```
```suggestion
*Description*: In Red Hat Process Automation Manager, a _rule_ provides the logic against which the rule engine executes. A rule includes a name, attributes, a “when” statement on the left side of the rule, and a “then” statement on the right side of the rule.
```
- Consider rephrasing the awkward syntax in the first sentence.

bburt-rh commented on supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc:
```diff
@@ -1114,11 +1104,11 @@ Use the name of the tab when you refer to it, for example, "the *Storage* tab".
 [[runtime-manager]]
 ==== image:images/yes.png[yes] runtime manager (noun)
-*Description*: In Red Hat JBoss BRMS and Red Hat JBoss BPM Suite, the _runtime manager_ is an interface that enables and simplifies the usage of a KIE API within the processes. The name of the interface is `RuntimeManager`. It provides configurable strategies that control actual runtime execution. The strategies are singleton, per request, and per process instance.
+*Description*: In Red Hat Process Automation Manager, the _runtime manager_ is an interface that enables and simplifies the usage of a KIE API within the processes. The name of the interface is `RuntimeManager`. It provides configurable strategies that control actual runtime execution. The strategies are singleton, per request, and per process instance.
```
```suggestion
*Description*: In Red Hat Process Automation Manager, the _runtime manager_ interface enables and simplifies the use of a KIE API within processes. The name of the interface is `RuntimeManager`. It provides configurable strategies that control actual runtime execution. The strategies are singleton, per request, and per process instance.
```
- Consider rephrasing to avoid awkward syntax.

bburt-rh commented on supplementary_style_guide/glossary_terms_conventions/general_conventions/s.adoc:
```diff
@@ -67,11 +67,11 @@
 [[scorecard]]
 ==== image:images/yes.png[yes] Scorecard (noun)
-*Description*: In Red Hat JBoss BRMS and Red Hat JBoss BPM Suite, _Scorecard_ is a risk management tool that is a graphical representation of a formula used to calculate an overall score. It is mostly used by financial institutions or banks to calculate the risk they can take to sell a product in the market. It can predict the likelihood or probability of a certain outcome. Red Hat JBoss BRMS sup ...
+*Description*: In Red Hat Process Automation Manager, _Scorecard_ is a risk management tool that is a graphical representation of a formula used to calculate an overall score. It is mostly used by financial institutions or banks to calculate the risk they can take to sell a product in the market. It can predict the likelihood or probability of a certain outcome. Red Hat Process Automation Manager ...
```
```suggestion
*Description*: In Red Hat Process Automation Manager, _Scorecard_ is a risk management tool that graphically represents a formula used to calculate an overall score. It is mostly used by financial institutions or banks to calculate the risk they take when selling a product. Scorecards help predict the likelihood or probability of a certain outcome. Red Hat Process Automation Manager supports additive scorecards that calculates an overall score by adding all partial scores assigned to individual rule conditions.
```
- Consider editing for clarity.

bburt-rh commented:
Added a few questions and suggested changes. PTAL and let me know if you have any questions.

joaedwar commented:
Hi @emmurphy1. I think this is close to being ready to merge. If you have time, could you please resolve the conflict and address the outstanding comments?

bergerhoffer commented on supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc:
```diff
@@ -513,6 +490,19 @@ Always spell out the full product name of the host, and do not capitalize the te
 *See also*: xref:red-hat-enterprise-linux-openstack-platform[Red Hat Enterprise Linux OpenStack Platform]
 
+// BxMS: General; kept as is
+[discrete]
```
@emmurphy1 can you please remove this [discrete] tag in this PR? We no longer use this in our glossary entries (see #365 for details)

mportman12 commented on supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc:
```diff
@@ -508,6 +472,17 @@ Always use the approved product name _Red Hat build of OpenJDK_ instead.
 
+[[red-hat-process-automation-manager]]
+==== image:images/yes.png[yes] Red Hat Process Automation Manager (noun)
+*Description*: _Red Hat Process Automation Manager_ is the Red Hat platform for Business Process Management (BPM). Bu using Red Hat Process Automation Manager, enterprise business and IT users can document, simulate, manage, automate, and monitor business processes and policies. The platform is designed to empower business and IT users to collaborate more effectively, so business applications can ...
```
```suggestion
*Description*: _Red Hat Process Automation Manager_ is the Red Hat platform for Business Process Management (BPM). Bu using Red Hat Process Automation Manager, enterprise business and IT users can document, simulate, manage, automate, and monitor business processes and policies.
```
The last sentence is pretty marketing-focused, and not needed in an SSG glossary entry.

mportman12 commented on supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc:
```diff
@@ -1027,21 +1002,21 @@ Use the name of the tab when you refer to it, for example, "the *Storage* tab".
 [[rule]]
 ==== image:images/yes.png[yes] rule (noun)
-*Description*: In Red Hat JBoss BRMS and Red Hat JBoss BPM Suite, a _rule_ provides the logic for the rule engine to execute against. A rule includes a name, attributes, a “when” statement on the left side of the rule, and a “then” statement on the right side of the rule.
+*Description*: In Red Hat Process Automation Manager, a _rule_ provides the logic against which the rule engine to executes. A rule includes a name, attributes, a “when” statement on the left side of the rule, and a “then” statement on the right side of the rule.
```
```suggestion
*Description*: In Red Hat Process Automation Manager, a _rule_ provides the logic that the rule engine executes against. A rule includes a name, attributes, a “when” statement on the left side of the rule that defines the rule conditions, and a “then” statement on the right side of the rule that defines the rule results.
```

mportman12 reviewed (changes requested):
Left a couple of suggestions.

mportman12 commented:
@emmurphy1 Please review the suggested changes when you have a chance.

mportman12 commented:
@emmurphy1 Please review the suggested changes.

emmurphy1 commented:
There were so many merge conflicts that it was easier to create a new branch and PR:
https://github.com/redhat-documentation/supplementary-style-guide/pull/453

I have addressed all of the comments here in that PR, and closing this PR.
